.gitignore

```diff
@@ -19,3 +20,6 @@ pip-log.txt
 # paste config files
 development.ini
 production.ini
+
+# Rope
+.ropeproject
```

Comment:
What's this needed for?

Reply:
Me.

Reply:
Yuck! _Please just add to your own global gitignore!!!_

No reason to clutter up our repository with harmful crap like this!!!

Reply:
Done.